js/utility/ChkBtnAction.js

```diff
@@ -0,0 +1,44 @@
+import { CLICK_EVENT, EDIT, COMPLETE, DELETE_TODO } from "/js/constants.js";
+import markTodoAsCompleted from "/js/markDone.js";
+import createErrorMessage from "/js/utility/errorMessage.js";
+
+export default function handleCheckButtonClick(
+  todoItemElement,
+  taskList,
+  id,
+  saveButtonElement,
+  deleteButtonElement,
+  completeButtonElement
+) {
+  function handleCheckButton(event) {
+    event.stopPropagation();
+    event.preventDefault();
+    const errorMessage = todoItemElement.querySelector("#error-message");
+    const textElement = todoItemElement.querySelector(".text");
+    const inputValue = textElement.innerText;
+
+    if (!inputValue) {
+      if (!errorMessage) {
+        const errorMessageCreated = createErrorMessage();
+        todoItemElement.appendChild(errorMessageCreated);
+      }
+      todoItemElement.querySelector(".text").focus();
+    } else {
+      if (errorMessage) {
+        todoItemElement.removeChild(errorMessage);
+      }
+      textElement.contentEditable = false;
+      markTodoAsCompleted(todoItemElement, taskList, id);
+      todoItemElement.querySelector(
+        `[data-job="${DELETE_TODO}"]`
+      ).style.display = "inline-block";
+      todoItemElement.querySelector(`[data-job="${EDIT}"]`).style.display =
+        "none";
+      saveButtonElement.style.display = "none";
+      completeButtonElement.style.display = "none";
+      deleteButtonElement.style.display = "none";
+      completeButtonElement.removeEventListener(CLICK_EVENT, handleCheckButton);
```

Comment:
```suggestion
      markTodoAsCompleted(todoItemElement, taskList, id);
      
      const deleteButton = todoItemElement.querySelector(`[data-job="${DELETE_TODO}"]`);
      const editButton = todoItemElement.querySelector(`[data-job="${EDIT}"]`);
      const buttonsToHide = [saveButtonElement, completeButtonElement, deleteButtonElement];
      buttonsToHide.forEach(button => button.style.display = "none");
      completeButtonElement.removeEventListener(CLICK_EVENT, handleCheckButton);
      editButton.style.display = "none";
      deleteButton.style.display = "inline-block";
```

Reply:
I have created [hideButtonByAttributes](https://github.com/ban-wtag/ToDoAppJS/pull/4#discussion_r1170851669) function and displayButtonByAttributes function to handle display styling. please check.